caching-service-package/src/main/resources/bin/start.sh

```diff
@@ -241,6 +241,7 @@ _BPX_JOBNAME=${ZWE_zowe_job_prefix}${CACHING_CODE} ${JAVA_BIN_DIR}java \
   -Dspring.profiles.active=${ZWE_configs_spring_profiles_active:-} \
   -Dspring.profiles.include=$LOG_LEVEL \
   -Dapiml.logs.location=${ZWE_zowe_logDirectory} \
+  -Dapiml.logging.timezone=${ZWE_zowe_logging_timezone} \
```

Comment:
Please provide a default value here and the the other `start.sh` scripts.
```suggestion
  -Dapiml.logging.timezone=${ZWE_zowe_logging_timezone:UTC} \
```